elixir/apps/domain/lib/domain/auth/adapters/google_workspace/api_client.ex

```diff
@@ -34,6 +34,50 @@ defmodule Domain.Auth.Adapters.GoogleWorkspace.APIClient do
     [conn_opts: [transport_opts: transport_opts]]
   end
 
+  # curl -d 'grant_type=urn%3Aietf%3Aparams%3Aoauth%3Agrant-type%3Ajwt-bearer&assertion=eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJodHRwczovL29hdXRoMi5nb29nbGVhcGlzLmNvbS90b2tlbiIsImV4cCI6MTcyNDE5NTQ5MCwiaWF0IjoxNzI0MTkxODkwLCJpc3MiOiJmaXJlem9uZS1pZHAtc3luY0BvcGVuaWQtY29ubmVjdC10ZXN0LTM5MTcxOS5pYW0uZ3NlcnZpY2VhY2NvdW50LmNvbSIsInNjb3BlIjoib3BlbmlkIGVtYWlsIHByb2ZpbGUgaHR0cHM6Ly93d3cuZ29vZ2xlYXBpcy5jb20vYXV0aC9hZG1pbi5kaXJlY3RvcnkuY3VzdG9tZXIucmVhZG9ubHkgaHR0cHM6Ly93d3cuZ29vZ2xlYXBpcy5jb20vYXV0aC9hZG1pbi5kaXJlY3Rvcnkub3JndW5pdC5yZWFkb25seSBodHRwczovL3d3dy5nb29nbGVhcGlzLmNvbS9hdXRoL2FkbWluLmRpcmVjdG9yeS5ncm91cC5yZWFkb25seSBodHRwczovL3d3dy5nb29nbGVhcGlzLmNvbS9hdXRoL2FkbWluLmRpcmVjdG9yeS51c2VyLnJlYWRvbmx5In0.hFZl772n74blSgJ2dsZiLKr72tZsDFazuLQ4dVEWBGeR-EEtrhkCA5D7FFuKSWQiW9fbu63LQlO-YiqICSuZv0BGkVzzfZ2Gs96PbGf5ezagq3SJA8H_pbo95Mmd2J_cLwn97NmlfXaksdSQgGdhGi_NtUFRavR_A7idvZWQIQyyt96l5eZULU1QeEm61zx6QFHvx8HiuYYqWurBbeg9_quiSsHJ2yXWRCAg98bEBu0swCV68uBzjkaQuAE-zCqnoDQdfVZCf4-11qoTs-_mgUURW86JhGAWjSKbrDCtHteprUgYRuhpxRsOt4TeK0cbIQFTP1eHxNOmOGfupM34fQ' https://oauth2.googleapis.com/token
```

Comment:
Guessing this was a debugging comment accidentally left in?

Reply:
Good catch, thanks